omnibus-snaxchain-testnet.toml

```
@@ -0,0 +1,29 @@
+name = "synthetix-omnibus"
+version = "1"
+description = "Snaxchain Testnet deployment"
+include = [
+    # Governance
+    "tomls/governance/councils-mothership.test.toml",
+    "tomls/governance/wormhole-testnet.toml",
+    "tomls/governance/snapshot-record.test.toml",
```

Please instead of adding suffixes `.test` or `-testnet`, move configs into subfolder matchint top level toml name `omnibus-snaxchain-testnet`

```suggestion
    "tomls/omnibus-snaxchain-testnet/governance/councils-mothership.toml",
    "tomls/omnibus-snaxchain-testnet/governance/wormhole.toml",
    "tomls/omnibus-snaxchain-testnet/governance/snapshot-record.toml",
```

---

same is for mainnet. All the included tomls should come from `tomls/omnibus-snaxchain/governance` subfolder and not from `tomls/governance`

---

fixed in #370